Tests/CasePathsTests/XCTUnwrapTests.swift

```diff
@@ -0,0 +1,30 @@
+import CasePaths
+import XCTest
+
+final class XCTUnwrapTests: XCTestCase {
```

Comment:
Should this test case be wrapped in `#if DEBUG` since it relies on overlay?

Reply:
yep good catch!